<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateTimeEditor</class>
 <widget class="QDialog" name="DateTimeEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DateTimeEditor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QCalendarWidget" name="calendarWidget"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="LeadingZerosSpinBox" name="sp_h">
       <property name="maximum">
        <number>23</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LeadingZerosSpinBox" name="sp_m">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pb_now">
     <property name="text">
      <string>&amp;Now</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LeadingZerosSpinBox</class>
   <extends>QSpinBox</extends>
   <header>leadingzerosspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sp_h</tabstop>
  <tabstop>sp_m</tabstop>
  <tabstop>pb_now</tabstop>
  <tabstop>calendarWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DateTimeEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DateTimeEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
